std/functional.d

```diff
@@ -26,6 +26,9 @@ $(TR $(TH Function Name) $(TH Description)
     $(TR $(TD $(D $(LREF lessThan)), $(D $(LREF greaterThan)), $(D $(LREF equalTo)))
         $(TD Ready-made predicate functions to compare two values.
     ))
+    $(TR $(TD $(D $(LREF lessOrEqualThan)), $(D $(LREF greaterOrEqualThan)), $(D $(LREF notEqualTo)))
```

Comment:
Why two different sections? They have the same descriptions, why not merge them?